themes/godotengine/pages/download/windows.htm

```diff
@@ -12,25 +12,25 @@
 {% put downloads %}
   <div class="btn split download">
     <a href="https://downloads.tuxfamily.org/godotengine/{{stable_version}}/Godot_v{{stable_version}}-stable_win64.exe.zip">
-      64-bit
+      64-bit (x86)
     </a>
   </div>
   <div class="btn split download">
     <a href="https://downloads.tuxfamily.org/godotengine/{{stable_version}}/Godot_v{{stable_version}}-stable_win32.exe.zip">
-      32-bit
+      32-bit (x86)
     </a>
   </div>
 {% endput %}
 
 {% put mono_downloads %}
   <div class="btn split download">
     <a href="https://downloads.tuxfamily.org/godotengine/{{stable_version}}/mono/Godot_v{{stable_version}}-stable_mono_win64.zip">
-      64-bit
+      64-bit (x86)
```

Comment:
I'm confused. I thought 32-bit was x86? And 64-bit was x64 (or x86_64)? Or do the ##-bit and x## terms mean different things despite being used interchangibly?

Reply:
Yeah might be worth fully specifying `x86_64` for 64-bit x86. That's what we use as binary extension on Linux.